cairo/ethereum/crypto/hash.cairo

```diff
@@ -11,6 +11,10 @@ EMPTY_ROOT:
 dw 0x6ef8c092e64583ffa655cc1b171fe856;  // low
```

Comment:
just realized I duplicated this value in `trie_diff.cairo` "EMPTY_TRIE_HASH_LOW" and "EMPTY_TRIE_HASH_HIGH"